### PR TITLE
virttest/remote.py: Add new param for RemoteFile.add()

### PR DESCRIPTION
**Pull request by PandaWei**
Add new param 'newline' to indicate the content of file
ends up with newline or not.

Signed-off-by: Wei,Jiangang <weijg.fnst@cn.fujitsu.com>

### CODE REVIEW

File: virttest/remote.py [hunk 1]
```diff
@@ -996,13 +996,17 @@ def _write_local(self, lines):
         local_file.writelines(lines)
         local_file.close()
 
-    def add(self, line_list):
+    def add(self, line_list, newline=False):
         """
         Append lines in line_list into file on remote.
+
+        :param newline: end up with newline or not
```
Comment (waynesun09) [inline virttest/remote.py]: since you updated here, better add another for 'line_list'

Comment (PandaWei) [inline virttest/remote.py]: Thanks @waynesun09 
Update it as your comments.

**Comment by waynesun09** [comment]
ACK

File: virttest/remote.py [hunk 2]
```diff
@@ -996,13 +996,18 @@ def _write_local(self, lines):
         local_file.writelines(lines)
         local_file.close()
 
-    def add(self, line_list):
+    def add(self, line_list, newline=False):
```
Comment (will-Do) [inline virttest/remote.py]: I prefer to use `linesep="\n"` , like all the using in aexpect.py module

Comment (PandaWei) [inline virttest/remote.py]: Yes, sounds reasonable.
Updated as comments

**Comment by lmr** [comment]
Hi @PandaWei. As you might know, we're moving virt-test to avocado-vt, so I please ask you to consider re-implementing this functionality in the later. The new repo is:

https://github.com/avocado-framework/avocado-vt

Please keep in mind the following guidelines when creating the new PR:

1) Most of the time you can pick the patches of your original PR and merge them on a new branch with minimal changes.
3) Don't use autotest APIs, since we're moving away from using them. Please review the code and change things like `utils.system` -> `process.system`, `utils.run` -> `process.run`, so on and so forth.

Thanks, and I'm sorry for any inconveniences we might have caused your team.

**Pull request closed without merging**